src/mrsimulator/spin_system/isotope.py

```diff
@@ -103,6 +116,64 @@ def larmor_freq(self, B0=9.4):
         return -self.gyromagnetic_ratio * B0
 
 
+def add_custom_isotope(
+    symbol: str,
+    spin: float,
+    gyromagnetic_ratio: float,
+    quadrupole_moment: float = 0,
+    natural_abundance: float = 100,
+    atomic_number: int = -1,  # What to put for this value??
+):
```

Comment:
Can this be a classmethod of `Isotope` class? Something like, `Isotope.add_new()`!